gallery/gallery-library/src/main/java/org/silverpeas/components/gallery/model/InternalMedia.java

```diff
@@ -158,13 +159,33 @@ public String getApplicationThumbnailUrl(MediaResolution mediaResolution) {
 
   @Override
   public SilverpeasFile getFile(final MediaResolution mediaResolution) {
-    String origOrThumbFileName = getFileName();
-    if (getType().isPhoto() && StringUtil.isDefined(mediaResolution.getThumbnailSuffix())) {
-      origOrThumbFileName = getId() + mediaResolution.getThumbnailSuffix();
+    if (StringUtil.isNotDefined(getFileName())) {
+      return SilverpeasFile.NO_FILE;
+    }
+    List<String> fileNames = new ArrayList<String>(2);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 2 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS109)